### ISSUE TITLE
[3.x]: Total Revenue Widget Includes Refunded Orders

### ISSUE THREAD

**Issue by philipboomy** [craftcms/commerce#2852]
### Description
Total Revenue widget is including refunded orders where paid status is unpaid. I believe that refunded orders should not be part of the revenue, similar to not reporting refunded orders in revenue in financial reports. I asked in Discord what peoples thoughts were on this and 1 replied that it shouldn't include it and another said there are times where unpaid orders should be included. I just feel generally speaking they should be included. Perhaps a setting could be implemented to remove/add unpaid orders.

### Craft CMS version

3.7.44

### Craft Commerce version

3.4.15

### Installed plugins and versions

-

**Comment by zackspear** [comment]
I support a change related to this. I recently released a bug that allowed 6000+ orders to go through for free (DOH!). I've since cleaned those orders up but our Revenue Widgets doesn't reflect my clean up.

Looking at the source code for how the revenue stat is tabulated the order `total` is being used.
https://github.com/craftcms/commerce/blob/develop/src/stats/TotalRevenue.php#L33

What could sort this out is using order `totalPaid` instead of `total` which should then show the amount of money that came in.
